vuepress/docs/next/tutorials/create/pb/export-bundle-from-application.md

```diff
@@ -71,14 +66,34 @@ mkdir bundle
 ```
 2. Run the bundler:
 ``` sh
-ent bundler from-env --location bundle --code YOUR-TEST-BUNDLE --description “Your Test Bundle”
+ent bundler from-env --location bundle --code YOUR-BUNDLE-NAME --description “Your Exported Bundle”
 ```
-The bundler will inspect the application using Entando APIs, collect information about individual components, construct appropriate descriptor files and assemble the top-level descriptor file.
+The bundler will inspect the application using Entando APIs, collect information about individual components, construct appropriate descriptor files, and assemble the top-level descriptor file.
 
 ``` 
 $ ls bundle
 assets      contentModels  contents         fragments  labels     pageModels  resources
 categories  contentTypes   descriptor.yaml  groups     languages  pages       widgets
 ```
 
-You now have a complete Entando project structure! You can inspect the output to edit the exported components or deploy it to another Entando Application.
+You now have a complete Entando project structure for a git-based bundle! You can inspect the output to edit the exported components or [deploy](publish-project-bundle.md) it to another Entando Application. 
+::: tip Note:
+To convert this project to a docker-based bundle, continue with the steps below. 
+:::
+
+### Construct the Docker-based Bundle Structure
+
+1. Initialize a new bundle and switch to that directory:
+``` 
+ent bundle init YOUR-BUNDLE-NAME
+cd YOUR-BUNDLE-NAME
+```
+2. Copy the resources from the bundle directory, with the exception of microservices (/plugins) and micro frontends (/widgets), to the `platform` directory:
+```
+cp -R ../testBundle/bundle/assets platform/
+```
+Repeat for all the other components in your bundle, including assets, contentModels, contents, fragments, labels, pageModels, resources, categories, contentTypes, groups, languages, and pages.
+
+If there are micro frontends or microservices in the bundle, use the source code to migrate them to their designated folders. Use the [ent bundle CLI tool](../../../docs/getting-started/ent-bundle.md) to assist in the process. Also check out the [Entando Bundle details](../../../docs/curate/bundle-details.md) page to define specific attributes for the MFEs and MSs in the bundle descriptor, `entando.json`.
+
+3. With the project structure in place, [build and publish](publish-project-bundle.md) your bundle.
```

Comment:
build and publish -> build and deploy (our usual phrasing when referring to this process and a bit more representative of what's occurring)

Reply:
in general i agree, but deploy is a more appropriate word imo. the end goal isn't to publish things to git/docker